tests/test_edr.py

```diff
@@ -39,6 +47,48 @@
 EDR_BLOCKS = os.path.join(DATA_DIR, 'blocks.edr')
 EDR_BLOCKS_XVG = os.path.join(DATA_DIR, 'blocks.xvg')
 
+# Testfiles for file version 1 with single precision
+# See GROMACS regressiontests `Initial revision` (96c57f0d)
+# http://redmine.gromacs.org/projects/regressiontests
+EDR_V1 = os.path.join(DATA_DIR, '1.edr')
+EDR_V1_XVG = os.path.join(DATA_DIR, '1.xvg')
+
+# Testfiles for file version 1 with double precision
+# See GROMACS regressiontests `Initial revision` (96c57f0d)
+# http://redmine.gromacs.org/projects/regressiontests
+EDR_V1_DOUBLE = os.path.join(DATA_DIR, '1_d.edr')
+EDR_V1_DOUBLE_XVG = os.path.join(DATA_DIR, '1_d.xvg')
+
+# Testfiles for file version 2
+# Generated from GROMACS regression test 'simple/imp1'
+# at branch 'release-4-5'
+# See https://gitlab.com/gromacs/gromacs-regressiontests
+# GROMACS version commit bcbfcdee8e449344605552fa90c18eeab2b1fc53
+EDR_V2 = os.path.join(DATA_DIR, '2.edr')
+EDR_V2_XVG = os.path.join(DATA_DIR, '2.xvg')
+EDR_V2_DOUBLE = os.path.join(DATA_DIR, '2_d.edr')
+EDR_V2_DOUBLE_XVG = os.path.join(DATA_DIR, '2_d.xvg')
+
+# Testfiles for file version 3
+# Generated from GROMACS regression test 'simple/imp1'
+# at branch 'release-4-5'
+# See https://gitlab.com/gromacs/gromacs-regressiontests
+# GROMACS version commit d9c1da8c98ef7a99db5de71c57f683cf19435ef5
+EDR_V3 = os.path.join(DATA_DIR, '3.edr')
+EDR_V3_XVG = os.path.join(DATA_DIR, '3.xvg')
+EDR_V3_DOUBLE = os.path.join(DATA_DIR, '3_d.edr')
+EDR_V3_DOUBLE_XVG = os.path.join(DATA_DIR, '3_d.xvg')
+
+# Testfiles for file version 3
```

Comment:
This should be `# Testfiles for file version 4`

Reply:
Oh, too much copy pasta...